### PR TITLE
security update for vsc-mympirun 4.1.9

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)
see https://github.com/hpcugent/vsc-base/security/advisories/GHSA-w8wc-fcgf-qf44

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node353.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/efd1a7049ab9888ca3a297206b94f279 for a full test report.

**Comment by boegel** [comment]
@smoors Strictly speaking this is not necessary, because `mympirun` doesn't use the `vsc.utils.rest` module, but it doesn't hurt either I guess...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2455.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/fb0fb00d8dd868c48a22f106a5b60888 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3108.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/2d4a064dd9e364f8e89fc2e852de2247 for a full test report.

File: easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.1.9.eb [hunk 1]
```diff
@@ -46,13 +46,12 @@ exts_list = [
...
     }),
-    ('vsc-base', '2.9.2', {
+    ('vsc-base', '2.9.6', {
         'modulename': 'vsc.utils',
         'source_urls': ['https://pypi.python.org/packages/source/v/vsc-base'],
-        'checksums': ['55803ce00f29bc0eb8b4e37518b21368b4c0f4cdb37c3a168912fc815fd88f15'],
+        'checksums': ['a62158022c36b5b456771ee8145d9ea3d299b301240e9724d04052e6aa0400b3'],
     }),
     (name, version, {
-        # don't check for import of vsc.mympirun, since that may fail if vsc-base is picked up from the OS
```
Comment (boegel) [inline easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.1.9.eb]: @smoors please restore this comment?

Comment (smoors) [reply easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.1.9.eb]: @boegel I actually did not delete it, it was `eb --inject-checksums` who did it :)

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**